.changeset/many-deers-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'sku': major
+---
+
+`sourceMapsProd` is now `true` by default
```

Comment:
I'd make it REALLY easy for someone going through a breaking change to see how they can resolve breaks:

```suggestion
`sourceMapsProd` is now `true` by default

To return to the previous behaviour set `sourceMapsProd: false` in your sku.config.js
```